Library/Formula/subversion.rb

```diff
@@ -70,7 +74,12 @@ class Subversion < Formula
   end
 
   def install
-    serf_prefix = libexec+"serf"
+    # Fixes: "libtool: error: error: relink 'libsvn_ra_serf-1.la'
+    # with the above command before installing it"
+    # Please see https://github.com/Homebrew/linuxbrew/issues/408 for details
+    ENV.deparallelize
```

Comment:
If you have access to a Mac OS machine, can you attempt to duplicate this bug on Mac OS? If it duplicates, can you send this chunk upstream to Homebrew/homebrew?